### PR TITLE
Milestone 3: Query multiple tables

### PR DESCRIPTION
**Pull request by sanieni6**
# Milestone 3: Query multiple tables
Hi Mr. Reviewer, In this part of the project I created new tables and made new queries.
## Summary:
+ Create a table named owners:

+ Create a table named species:

+ Modify animals table:

+ Insert the new data into the owners table:

+ Insert the new data into the species table:

+ Modify your inserted animals so it includes the species_id value:

+ Modify your inserted animals to include owner information (owner_id):

### Queries
+ What animals belong to Melody Pond?
image
+ List of all animals that are pokemon (their type is Pokemon).
image
+ List all owners and their animals, remember to include those that don't own any animal.
image
+ How many animals are there per species?
image

+ List all Digimon owned by Jennifer Orwell.
image

+ List all animals owned by Dean Winchester that haven't tried to escape.
image

+ Who owns the most animals?
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -12,3 +12,24 @@ CREATE TABLE animals (
...
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) 
+);
+
+ALTER TABLE animals ALTER COLUMN id SET NOT NULL;
+ALTER TABLE animals ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY;
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD COLUMN species_id INT;
```
Comment (Shedrack-Sunday) [inline schema.sql]: - [ ] This query should reference `species` table

## Sample code:
``ALTER TABLE animals ADD species_id int REFERENCES species(id);``

File: schema.sql [hunk 2]
```diff
@@ -12,3 +12,24 @@ CREATE TABLE animals (
...
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id);
+ALTER TABLE animals ADD COLUMN owner_id INT;
```
Comment (Shedrack-Sunday) [inline schema.sql]: - [ ] This query should reference the `owners` table

## Sample code:
`ALTER TABLE animals ADD owner_id int REFERENCES owners(id)`

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @sanieni6        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Rights file structure✔️  
- Good use SQL queries ✔️
   
  
# Required Changes  🌵 

- [ ] There are a few adjustments you need to make to make your work perfect. 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by sanieni6** [comment]
hi @Shedrack-Sunday, thanks for your review...
I'm using these command for do that:
https://github.com/sanieni6/Vet-clinic-database/blob/8436639e2691d0c51e684e6040d5db236692037d/schema.sql#L32
https://github.com/sanieni6/Vet-clinic-database/blob/8436639e2691d0c51e684e6040d5db236692037d/schema.sql#L34
Don't these commands do the same?
image

**Comment by sanieni6** [comment]
# Changes applied
+ Modify the references
https://github.com/sanieni6/Vet-clinic-database/blob/faf4cf8854169245e8d04edc5c054411707e477b/schema.sql#L31-L32

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @sanieni6, 

To highlight 👍🏽 
✔️ Queries are added
✔️ Screenshots are added

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**